modules/json/src/json-loader.js

```diff
@@ -12,7 +12,7 @@ export const JSONLoader = {
   id: 'json',
   name: 'JSON',
   version: VERSION,
-  extensions: ['json'],
+  extensions: ['json', 'geojson'],
```

Comment:
We also have a separate geojson loader now, allowing for geojson to be parsed into binary. And we don't have a good system for handling overrides yet. @kylebarron

Reply:
I think the point of having a `GeoJSONLoader` is so that we don't have to include `geojson` here, but know we can treat the output as geometries instead of arbitrary `json`.

Reply:
Yes. I think it would be really good to have a way to install multiple loaders for a format such that the application can control the priority. We should really try to get that in before loaders.gl 2.2 code freeze end of this week.

Reply:
This is only a problem when you use `registerLoaders`, right? If you use a loader explicitly, you can define either of the following I assume
```
load(jsonUrl, JsonLoader)
load(jsonUrl, GeoJsonLoader)
```

Reply:
Yes, I can specify GeoJsonLoader. The question remain...will GeoJSONLoader sends back `progress` in batches?